lib/build.gradle

```diff
@@ -14,7 +18,7 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
         }
     }
-
+    testOptions.unitTests.includeAndroidResources = true
```

Comment:
Why this change is required?

Reply:
Hi @epool ,

There were warning in test output about legacy resource mode that is apparently deprecated.:
```
[Robolectric] NOTICE: legacy resources mode is deprecated; see http://robolectric.org/migrating/#migrating-to-40
```

After adding this option output from tests is much more clean.

Does this answer Your question? 

Thanks!